src/MSALWrapper/AuthMode.cs

```diff
@@ -5,64 +5,40 @@ namespace Microsoft.Authentication.MSALWrapper
 {
     using System;
 
-    /// <summary>
-    /// The auth mode.
-    /// </summary>
+    /// <summary>/// The auth mode./// </summary>
     [Flags]
     public enum AuthMode : short
     {
-        /// <summary>
-        /// The web.
-        /// </summary>
+        /// <summary>/// The web./// </summary>

```

Comment:
We want the summary to be more meaningful than "The web". These were added by me using resharper which does a poor job of creating meaingful summaries but adds a placeholder.